src/components/blocks/software/Code/InlineCodeElement.tsx

```diff
@@ -1,7 +1,10 @@
 import React from 'react';
 
 const InlineCodeElement = ({ children, ...props }: { children: React.ReactNode }) => (
-  <code {...props} className="font-mono font-semibold text-code p-4">
+  <code
+    {...props}
+    className="font-mono p-2 text-code text-charcoal-grey font-thin bg-light-grey border border-mid-grey rounded-md"
```

Comment:
thinking can it be just `ui-text-code-inline` ? then add  `font-thin` and `border`?

Reply:
Just tested this @aralovelace , doesn't work because there's priority conflicts between font-thin and ui-text-code-inline (which sets font-weight: 600).